src/com/estafet/learning/US008/TradeOrder.java

```diff
@@ -0,0 +1,15 @@
+package com.estafet.learning.US008;
+
+public class TradeOrder extends Order {
+
+    public TradeOrder(){
+
+    }
+    public void tradeOrder(){
+        final String color = "red";
```

Comment:
make it a class member instead of local var

Reply:
changed.